wagtailio/project_styleguide/templates/patterns/components/streamfields/quotes.html

```diff
@@ -0,0 +1,21 @@
+{% load wagtailcore_tags %}
+
+<div class="quotes">
+    <div class="quotes__header">
+        <p class="quotes__title heading-one">{{ value.title }}</p>
+        <div>
+            <svg aria-hidden="true" class="quotes__icon"><use xlink:href="#quotes"></use></svg>
+        </div>
+    </div>
+    <div class="quotes__grid">
```

Comment:
In the design the quotes align to the grid, which you can use the `grid` class for.

<img width="870" alt="image" src="https://user-images.githubusercontent.com/31627284/181790107-8dc1c735-a66a-4c72-8703-56bd35ab30f7.png">

Reply:
Thanks!